### PR TITLE
New Kudos: Polyglot Programmer

### PR DESCRIPTION
**Pull request by jakerockland**
I want to mint my own Kudos plx @owocki

# Title
Polyglot Programmer

# Design Inspiration
With a [Solarized](https://ethanschoonover.com/solarized/) color scheme, has excerpts of code in C++, JavaScript, Python, Java, PHP, Go, and Ruby!

# Metadata
name: Polyglot Programmer
description: For those who love languages, programming or otherwise!
priceFinney: 4
numClonesAllowed: 200
tags: programming, languages, c++, javascript, python, java, php, go, ruby

# Down Payment
Question: do I need to put a down payment if I made the art?

# Gen0 Owner Address
My mainnet address is: `0xF5D4020dCA6a62bB1efFcC9212AAF3c9819E30D7`

### CODE REVIEW

**Comment by jakerockland** [comment]
Grr, not quite ready for merge yet, something odd happening with my svg export.

**Comment by jakerockland** [comment]
Fixed 👍 😄

**Comment by jakerockland** [comment]
Fixed 🙌

**Comment by owocki** [comment]
Thanks.. Im working on a new kudos minting process.   If you don't mind waiting a few more days @jakerockland I can merge this then.

**Comment by jakerockland** [comment]
@owocki No rush at all 😄 I'm mostly just curious about testin the system and like participating in the ecosystem. No real time pressure on my end. 👍

**Comment by owocki** [comment]
@jakerockland can u add the metadata to the kudos.yaml file in the dir?

**Comment by jakerockland** [comment]
@owocki done!

File: kudos.yaml [hunk 1]
```diff
@@ -1984,5 +1984,18 @@
   tags: 
     - memes
   image: your_bounty_is_bad_and_you_should_fill_bad.svg
-
-
+- name: polyglot_programmer
+  description: For those who love languages, programming or otherwise!
```
Comment (owocki) [inline kudos.yaml]: @jakerockland can you put your address in there as `to_address` 
and also add a param called `artist` that has your github username in it. 

these params arent fully baked into the site yet.. (you cant see them on the frontend), but eventually when we build artist pages theyll matter

**Comment by owocki** [comment]
https://gitcoin.co/kudos/809/polyglot_programmer

**Pull request merged**